2016-12-19 13:56:47,322 INFO: 200 GET /admin/login/ (::1) 262.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 13:56:47,759 INFO: 200 GET /static/admin/css/style.css?v=5618a5595ec19f6a6e6bc342fa22d004 (::1) 378.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 13:56:48,707 WARNING: 404 GET /favicon.ico (::1) 40.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 14:12:21,000 INFO: 200 GET /admin/manage/ (::1) 1066.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 14:12:21,030 INFO: 200 GET /static/admin/css/light-custome.css?v=5938dc9c6856e940a7d67b26740ccbf9 (::1) 2.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 14:12:21,043 INFO: 200 GET /static/admin/js/manage/manageView.js?v=7203a454b6f3eb8a32f3781d4d540e71 (::1) 1.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:51:56,447 INFO: 200 GET /admin/login/ (::1) 27.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:52:02,608 INFO: 200 POST /admin/login/ (::1) 479.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:52:02,609 ERROR: Uncaught exception POST /admin/login/ (::1)
HTTPServerRequest(protocol='http', host='localhost:8000', method='POST', uri='/admin/login/', version='HTTP/1.1', remote_ip='::1', headers={'Origin': 'http://localhost:8000', 'Content-Length': '102', 'Accept-Language': 'zh-CN,zh;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1', 'Dnt': '1', 'Connection': 'keep-alive', 'Referer': 'http://localhost:8000/admin/login/', 'Cache-Control': 'max-age=0', 'Cookie': 'csrftoken=JHTXaFbn2qW0MSBlS4mD8lqJNj5ymREL; _xsrf=2|3575007d|e4f293b88eaa1880233827a1ae4c39e7|1480862861; username="2|1:0|10:1482061027|8:username|8:a2lkb3po|e96df24d9c204b9f4eb0d8aa6ba45851b828060495c78b82cc0b261978413248"', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36', 'Content-Type': 'application/x-www-form-urlencoded'}) [in D:\Canopy\User\lib\site-packages\tornado\web.py:1496]
Traceback (most recent call last):
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 1413, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "F:\python\tornado\Tjango\contrib\admin\view.py", line 91, in post
    self.redirect(urlMapper.get_url_by_handler('contrib.admin.view.adminManageHandler'))
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 664, in redirect
    raise Exception("Cannot redirect after headers have been written")
Exception: Cannot redirect after headers have been written
2016-12-19 16:55:26,957 INFO: 200 POST /admin/login/ (::1) 372.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:55:26,959 ERROR: Uncaught exception POST /admin/login/ (::1)
HTTPServerRequest(protocol='http', host='localhost:8000', method='POST', uri='/admin/login/', version='HTTP/1.1', remote_ip='::1', headers={'Origin': 'http://localhost:8000', 'Content-Length': '102', 'Accept-Language': 'zh-CN,zh;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1', 'Dnt': '1', 'Connection': 'keep-alive', 'Referer': 'http://localhost:8000/admin/login/', 'Cache-Control': 'max-age=0', 'Cookie': 'csrftoken=JHTXaFbn2qW0MSBlS4mD8lqJNj5ymREL; _xsrf=2|3575007d|e4f293b88eaa1880233827a1ae4c39e7|1480862861; username="2|1:0|10:1482162722|8:username|8:a2lkb3po|b98ab98384085250b794fe6a4e2409c22b89ce055a95dd264f462705a9bcc73c"', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36', 'Content-Type': 'application/x-www-form-urlencoded'}) [in D:\Canopy\User\lib\site-packages\tornado\web.py:1496]
Traceback (most recent call last):
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 1413, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "F:\python\tornado\Tjango\contrib\admin\view.py", line 91, in post
    else:
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 664, in redirect
    raise Exception("Cannot redirect after headers have been written")
Exception: Cannot redirect after headers have been written
2016-12-19 16:55:32,766 INFO: 200 POST /admin/login/ (::1) 381.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:55:32,767 ERROR: Uncaught exception POST /admin/login/ (::1)
HTTPServerRequest(protocol='http', host='localhost:8000', method='POST', uri='/admin/login/', version='HTTP/1.1', remote_ip='::1', headers={'Origin': 'http://localhost:8000', 'Content-Length': '102', 'Accept-Language': 'zh-CN,zh;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1', 'Dnt': '1', 'Connection': 'keep-alive', 'Referer': 'http://localhost:8000/admin/login/', 'Cache-Control': 'max-age=0', 'Cookie': 'csrftoken=JHTXaFbn2qW0MSBlS4mD8lqJNj5ymREL; _xsrf=2|3575007d|e4f293b88eaa1880233827a1ae4c39e7|1480862861; username="2|1:0|10:1482162926|8:username|8:a2lkb3po|09f28a39a07678da19ff6f1fe231a6fde5eb6ac99f6ab6c522f66e18774f65ce"', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36', 'Content-Type': 'application/x-www-form-urlencoded'}) [in D:\Canopy\User\lib\site-packages\tornado\web.py:1496]
Traceback (most recent call last):
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 1413, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "F:\python\tornado\Tjango\contrib\admin\view.py", line 91, in post
    else:
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 664, in redirect
    raise Exception("Cannot redirect after headers have been written")
Exception: Cannot redirect after headers have been written
2016-12-19 16:55:45,046 INFO: 302 POST /admin/login/ (::1) 362.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 16:55:45,048 ERROR: Uncaught exception POST /admin/login/ (::1)
HTTPServerRequest(protocol='http', host='localhost:8000', method='POST', uri='/admin/login/', version='HTTP/1.1', remote_ip='::1', headers={'Origin': 'http://localhost:8000', 'Content-Length': '102', 'Accept-Language': 'zh-CN,zh;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Host': 'localhost:8000', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8', 'Upgrade-Insecure-Requests': '1', 'Dnt': '1', 'Connection': 'keep-alive', 'Referer': 'http://localhost:8000/admin/login/', 'Cache-Control': 'max-age=0', 'Cookie': 'csrftoken=JHTXaFbn2qW0MSBlS4mD8lqJNj5ymREL; _xsrf=2|3575007d|e4f293b88eaa1880233827a1ae4c39e7|1480862861; username="2|1:0|10:1482162932|8:username|8:a2lkb3po|3a1a8f5dd17c75e26a4cf05b4c6dee6226aba8f1b7fb200988c1e5a8c477424e"', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36', 'Content-Type': 'application/x-www-form-urlencoded'}) [in D:\Canopy\User\lib\site-packages\tornado\web.py:1496]
Traceback (most recent call last):
  File "D:\Canopy\User\lib\site-packages\tornado\web.py", line 1413, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "F:\python\tornado\Tjango\contrib\admin\view.py", line 98, in post
    self.render('contrib/admin/login.html', **args)
TypeError: render() argument after ** must be a mapping, not tuple
2016-12-19 16:55:46,154 INFO: 200 GET /admin/manage/ (::1) 1088.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 17:05:37,954 INFO: F:\python\tornado\Tjango\contrib\admin\sysStatus.py modified; restarting server [in D:\Canopy\User\lib\site-packages\tornado\autoreload.py:204]
2016-12-19 21:26:32,178 INFO: 200 GET /admin/login/ (::1) 17.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 21:26:32,753 WARNING: 404 GET /favicon.ico (::1) 18.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
2016-12-19 22:28:34,223 WARNING: 404 GET /favicon.ico (::1) 6.00ms [in D:\Canopy\User\lib\site-packages\tornado\web.py:1908]
